kin-devplatform-sdk/src/main/java/kin/devplatform/data/blockchain/BlockchainSourceImpl.java

```diff
@@ -365,31 +366,31 @@ public void removePaymentObserver(Observer<Payment> observer) {
 
 	@Override
 	public void createTrustLine(@NonNull final KinCallback<Void> callback) {
-		new CreateTrustLineCall(account, new TrustlineCallback() {
-			@Override
-			public void onSuccess() {
-				eventLogger.send(StellarKinTrustlineSetupSucceeded.create());
-				mainThread.execute(new Runnable() {
-					@Override
-					public void run() {
-						callback.onResponse(null);
-					}
-				});
-			}
+			new CreateTrustLineCall(account, new TrustlineCallback() {
+				@Override
+				public void onSuccess() {
+					eventLogger.send(StellarKinTrustlineSetupSucceeded.create());
+					mainThread.execute(new Runnable() {
+						@Override
+						public void run() {
+							callback.onResponse(null);
+						}
+					});
+				}
 
-			@Override
-			public void onFailure(final OperationFailedException e) {
-				eventLogger.send(
-					StellarKinTrustlineSetupFailed.create(ErrorUtil.getPrintableStackTrace(e), "", e.getMessage()));
-				mainThread.execute(new Runnable() {
-					@Override
-					public void run() {
-						callback.onFailure(ErrorUtil.getBlockchainException(e));
-					}
-				});
-			}
-		}).start();
-	}
+				@Override
+				public void onFailure(final OperationFailedException e) {
+					eventLogger.send(
+						StellarKinTrustlineSetupFailed.create(ErrorUtil.getPrintableStackTrace(e), "", e.getMessage()));
+					mainThread.execute(new Runnable() {
+						@Override
+						public void run() {
+							callback.onFailure(ErrorUtil.getBlockchainException(e));
+						}
+					});
+				}
+			}).start();
+		}
```

Comment:
Same comment as above - format the code because i think you have an extra tab

Reply:
:+1: